Tests/XMLCoderTests/ErrorContextTest.swift

```diff
@@ -29,11 +29,19 @@ final class ErrorContextTest: XCTestCase {
                 return
             }
 
-            XCTAssertEqual(ctx.debugDescription, """
-            \(underlying.localizedDescription) \
-            at line 1, column 2:
-            `<blah `
-            """)
+            #if os(Linux)
+                XCTAssertEqual(ctx.debugDescription, """
+                \(underlying.localizedDescription) \
+                at line 1, column 7:
+                `ah //>`
+                """)
+            #else
+                XCTAssertEqual(ctx.debugDescription, """
+                \(underlying.localizedDescription) \
+                at line 1, column 2:
+                `<blah `
+                """)
```

Comment:
I don't think this special handling on Linux would be very reliable. I'd definitely prefer this inconsistency to be reported on bugs.swift.org before merging this PR. Links to those bug reports should be added as comments in relevant code and documentation, so that both users and developers are aware of this.